src/components/sidebar.tsx

```diff
@@ -245,12 +245,13 @@ export default () => {
 
         <ul className="list-unstyled" data-sidebar-tree>
           <SidebarLink to="/integrations/">Overview</SidebarLink>
+          <SidebarLink to="/integrations/azuredevops/">Azure DevOps</SidebarLink>
+          <SidebarLink to="/integrations/bitbucket/">Bitbucket</SidebarLink>
           <SidebarLink to="/integrations/github/">GitHub</SidebarLink>
+          <SidebarLink to="/integrations/gitlab/">GitLab</SidebarLink>
+          <SidebarLink to="/integrations/msteams/">Microsoft Teams</SidebarLink>
           <SidebarLink to="/integrations/slack/">Slack</SidebarLink>
           <SidebarLink to="/integrations/vercel/">Vercel</SidebarLink>
-          <SidebarLink to="/integrations/msteams/">Microsoft Teams</SidebarLink>
-          <SidebarLink to="/integrations/gitlab/">GitLab</SidebarLink>
-          <SidebarLink to="/integrations/bitbucket/">Bitbucket</SidebarLink>
```

Comment:
Love it!

I don't _think_ you can add a comment in a `return` statement but would be so nice if we could add a comment saying this is alphabetical!

Reply:
@manuzope `{/* A JSX comment */}` should work fine I believe

Reply:
Dang, nice. TIL!